spec/controllers/v1/item_types_controller_spec.rb

```diff
@@ -7,7 +7,7 @@
     let(:allowed_keys) { %w(color length) }
     let(:other_item_type) { create :item_type }
     context 'write access to another item_type' do
-      before(:each) { authenticate_with_access_to :write, other_item_type }
+      before(:each) { @service = authenticate_with_access_to :write, other_item_type }
```

Comment:
In general our Rails 4 apps don't use instance variables in tests. Better to use `let` blocks. The good news is `authenticate_with_access_to` [returns the service as which it authenticated you](https://github.com/umts/aggressive-epsilon/blob/d6138e640e1c9249c70118ab40e14d7d62312f43/spec/spec_helper.rb#L39), i.e. you can just do

```ruby
  let(:service) { authenticate_with_access_to :write, other_item_type }
```